SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-4028/SpatialAuthorityTestGameState.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "SpatialAuthorityTestGameState.h"
+#include "Components/SceneComponent.h"
```

Comment:
I might be wrong, but is this include needed? 
```suggestion

```

Reply:
I think you're right, will double check